tools/Run.cmd

```diff
@@ -0,0 +1,71 @@
+@echo off
+
+rem NOTE: Copy of the corresponding file form node-repo-init v3.0.0 repository
```

Comment:
Ditto, how about `rem NOTE: Copy of https://github.com/pandell/node-repo-init/blob/v3.0.0/sync/tools/Run.cmd`